be/src/runtime/tablets_channel.cpp

```diff
@@ -102,7 +111,7 @@ Status TabletsChannel::close(int sender_id, int64_t backend_id, bool* finished,
         _state = kFinished;
         // All senders are closed
         // 1. close all delta writers
-        std::vector<DeltaWriter*> need_wait_writers;
+        std::set<DeltaWriter*> need_wait_writers;
```

Comment:
Why change to std::set?

Reply:
> Why change to std::set?

Element erase operation will follow and  std::set is more efficient than std::vector to execute `erase`.